src/Globe/Globe.js

```diff
@@ -187,9 +186,11 @@ Globe.prototype.setLayerVisibility = function(id, visible) {
     this.layersConfiguration.setLayerVisibility(id, visible);
 
     var cO = function(object) {
```

Comment:
for now, our layers does not use this function. We cannot create a root in map and add layer dynamically to it. Our the layer is added into scene after its creation otherwise it does not work. Do you remenber the bugs i asked you before?